app/src/main/java/dev/andrylat/raqimbek/bankingutils/BankingUtilsApp.java

```diff
@@ -1,43 +1,116 @@
 package dev.andrylat.raqimbek.bankingutils;
 
-import java.util.Scanner;
-import java.util.List;
+import lombok.NonNull;
+
+import java.util.*;
 
 public class BankingUtilsApp {
-  private static final UserInteraction userInteraction =
-      new UserInteraction(new Scanner(System.in), System.out);
-  private static final CardValidator cardValidator = new CardValidator();
-  private static final PaymentSystemDeterminer paymentSystemDeterminer =
-      new PaymentSystemDeterminer();
+  private static final CommandLineUserInteraction commandLineUserInteraction =
+      new CommandLineUserInteraction(new Scanner(System.in), System.out);
 
   public static void main(String[] args) {
     run();
```

Comment:
Why is it done in this way? What is the problem to write the logic from method run() here - in the main()?